pkg/controller/iotconfig/file_device_registry.go

```diff
@@ -49,7 +49,7 @@ func (r *ReconcileIoTConfig) processFileDeviceRegistry(ctx context.Context, conf
 
 	if !util.IsOpenshift() {
 		rc.ProcessSimple(func() error {
-			return r.processService(ctx, nameDeviceRegistry + "-external", config, r.reconcileFileDeviceRegistryServiceExternal)
+			return r.processService(ctx, nameDeviceRegistry+"-external", config, r.reconcileFileDeviceRegistryServiceExternal)
```

Comment:
why ?????

Reply:
> This also includes a few formatting changes automatically applied by `go fmt`.

This is the way go formats its code. I would impose anything other than using `go fmt`.